snap_config

```diff
@@ -53,6 +53,9 @@ while [ -z "$SETUP_DONE" ]; do
     "AD8K5" )
       SNAP_CFG="CONFIG_AD8K5=y\n""CONFIG_FPGACHIP=xcku115-flva1517-2-e\n"
       ;;
+    "RCXVUP" )
```

Comment:
The file snap_config is not used anywhere and got removed

Reply:
removed